Explorer/Assets/DCL/Infrastructure/ProtobufPartialClasses/IDirtyMarker.cs

```diff
@@ -5,186 +5,223 @@ namespace DCL.ECSComponents
     public interface IDirtyMarker
     {
         bool IsDirty { get; set; }
+        bool DelayDirtyReset { get; set; }
```

Comment:
`DelayDirtyReset` appears to be highly individualized logic. I can propose an alternative: instead of adding it to the common interface, don't register `ResetDirtyFlagSystem<PBGltfContainer>.InjectToWorld(ref builder);` (thus you will avoid execution of the common logic) but reset that flag manually in `LoadGltfContainerSystem` (instead of forcing to keep it) - thus you will declare explicitly your intentions without generalizing the individual logic

Reply:
Cool, yeah that's definitely better, that's why I wanted you to take a look at this core change, thank you!
I'll try to make it work that way then.